api/server/models/user.js

```diff
@@ -378,7 +378,9 @@ module.exports = function (User) {
                                         var userId = value.id;
                                         var pic = value.profile_image;
                                         var lastSeen = value.last_seen;
-                                        userInfo.push({name: userName, id: userId, pic: pic, lastSeen: lastSeen});
+                                        if (access_token_userid != userId) {
+                                            userInfo.push({name: userName, id: userId, pic: pic, lastSeen: lastSeen});
```

Comment:
this is wrong. paging wont work properly with this. this should be removed from mongo query itself